src/main/java/com/flightstats/hub/app/ClusterHubBindings.java

```diff
@@ -50,7 +56,31 @@ protected void configure() {
         bind(S3BatchManager.class).asEagerSingleton();
         bind(S3Verifier.class).asEagerSingleton();
         bind(AppUrlCheck.class).asEagerSingleton();
-        bind(SpokeTtlEnforcer.class).asEagerSingleton();
+
+        bind(SpokeTtlEnforcer.class)
+                .annotatedWith(Names.named(SpokeStore.WRITE.name()))
+                .toInstance(new SpokeTtlEnforcer(
+                        HubProperties.getSpokePath(SpokeStore.WRITE),
+                        HubProperties.getSpokeTtlMinutes(SpokeStore.WRITE)));
+
+        bind(SpokeTtlEnforcer.class)
+                .annotatedWith(Names.named(SpokeStore.READ.name()))
+                .toInstance(new SpokeTtlEnforcer(
+                        HubProperties.getSpokePath(SpokeStore.READ),
+                        HubProperties.getSpokeTtlMinutes(SpokeStore.READ)));
+
+        bind(FileSpokeStore.class)
```

Comment:
Can the two FileSpokeStore bindings here and in SingleHubBindings be moved up to HubBindings?